### ISSUE TITLE
Named Pipes equivalent for linux?

### ISSUE THREAD

**Issue by Rachel-ResDiary** [Microsoft/Docker.DotNet#334]
I'm trying to run a .Net core 2.0 application that utilises Docker.DotNet in a Linux container. The line below works fine in Windows. But seems to cause issues in Linux:

```DockerClient localDockerDaemon = new DockerClientConfiguration(new Uri("npipe://./pipe/docker_engine")).CreateClient()```

I've tried doing this instead, as my understanding is that Named Pipes only apply to Docker For Windows:

```DockerClient localDockerDaemon = new DockerClientConfiguration(new Uri(@"file:/var/run/docker.sock")).CreateClient()```

But this doesn't work either. 

Any suggestions for the appropriate Uri to use when running in Linux?

**Comment by MartinPesek** [comment]
Hey,

this should work:
`var client = new DockerClientConfiguration(new Uri("unix:///var/run/docker.sock")).CreateClient();`

**Comment by Rachel-ResDiary** [comment]
@MartinPesek Thanks for that. Unfortunately that didn't work for me. Probably because I'm doing a weird thing with using a sidecar container to access docker. That 'docker.sock' file doesn't exist on my main container. In case this helps anyone else, what I ended up doing to fix it was I went into the dashboard and had a look at my main container. There was an Environment Variable in there that was set to the value I needed:

![2018-08-22 where to find docker host](https://user-images.githubusercontent.com/39724678/44464241-25e22580-a612-11e8-8355-2d0aeb40fcbc.png)

So, I ended up with:

```static DockerClient localDockerDaemon = new DockerClientConfiguration(new Uri(@"tcp://localhost:2375")).CreateClient();```
